### PR TITLE
among pequeno

### PR DESCRIPTION
**Pull request by EmoGarbage404**
## About the PR 
among pequeno
**Screenshots**

image

**Changelog**

:cl:
- add: Added a sussy lil friend to the station.

### CODE REVIEW

**Review by Emisse — APPROVED** [review]

**Comment by EmoGarbage404** [comment]
leaving as DNM until we get the letter from among HQ

**Review by Just-a-Unity-Dev — APPROVED** [review]

**Review by Veritius — APPROVED** [review]

**Comment by EmoGarbage404** [comment]
clarified copyright policy for sprites in the rsi

**Review by vulppine — APPROVED** [review]
**AMOUNG** (5.50 credits)

*amoung pequeño* (3.00 credits)

File: Resources/Prototypes/Entities/Objects/Fun/toys.yml [hunk 1]
```diff
@@ -702,3 +702,21 @@
       damage:
         types:
           Blunt: 0
+
+- type: entity
+  parent: BasePlushie
+  id: ToyAmongPequeno
+  name: among pequeno
```
Comment (moonheart08) [inline Resources/Prototypes/Entities/Objects/Fun/toys.yml:709]: Needs to have a static price of three credits.

File: Resources/Textures/Objects/Fun/toys.rsi/meta.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/commit/e1142f20f5e4661cb6845cfcf2dd69f864d67432, orb, orb-inhand-left and orb-inhand-right created by Pancake",
+  "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/commit/e1142f20f5e4661cb6845cfcf2dd69f864d67432, orb, orb-inhand-left and orb-inhand-right created by Pancake, among-base and among-visor used with respect to the Among Us IP policy at https://www.innersloth.com/fan-creation-policy/",
```
Comment (moonheart08) [inline Resources/Textures/Objects/Fun/toys.rsi/meta.json:4]: This is wrong, check your ping in shiptest for the correct copyright for the pequeno, and make a new RSI.

**Review by moonheart08 — CHANGES_REQUESTED** [review]

**Review by moonheart08 — APPROVED** [review]

**Pull request merged**